Docs/Configuration.tex

```diff
@@ -5996,20 +5994,20 @@ \subsection{APFS Properties}\label{uefiapfsprops}
   \textbf{Failsafe}: \texttt{false}\\
   \textbf{Description}: Load APFS drivers for newly connected devices.
 
-  Performs APFS driver loading not only at OpenCore startup but also
-  during the OpenCore picker. This permits APFS USB hot plug. Disable
-  if not required.
+  Permits APFS USB hot plug which enables loading APFS drivers, both at OpenCore
+  startup and during OpenCore picker dusplay. Disable if not required.
 
 \item
   \texttt{MinDate}\\
   \textbf{Type}: \texttt{plist\ integer}\\
   \textbf{Failsafe}: \texttt{0}\\
   \textbf{Description}: Minimal allowed APFS driver date.
 
-  APFS driver date connects APFS driver with the calendar
-  release date. Older versions of APFS drivers may contain unpatched
-  vulnerabilities, which can be used to inflict harm to the computer.
-  This option permits restricting APFS drivers to only recent releases.
+  The APFS driver date connects the APFS driver with the calendar elease date.
+  Apple ultimately drops support for APFS drivers from older macOS releases and
```

Comment:
It does not. Rather it drops support of old macOS versions. As a result apfs drivers stop getting updated.